CONTRIBUTING.md

```diff
@@ -5,6 +5,25 @@
 
 _See also: [Flutter's code of conduct](https://github.com/flutter/flutter/blob/master/CODE_OF_CONDUCT.md)_
 
+## Important note
+
+As of January 2021, we are no longer accepting non-critical PRs for plugins
```

Comment:
Do we have an example or explanation of what a "non-critical" PR is? Or is it just PRs for all the plugins in the list below?

Reply:
Good point, reworded below to give examples of "critical".

(I expect we'll get some amount of pushback regardless, with people arguing that, e.g., a specific feature is critical to them, but this should help better set expectations.)